src/ActionModal/components/ModalWithdrawRewards.vue

```diff
@@ -76,6 +79,18 @@ export default {
     },
     validatorsWithRewards() {
       return this.rewards.length > 0
+    },
+    claimedReward() {
+      if (this.denom && this.rewards && this.rewards.length > 0) {
+        // we return the staking denom reward if it has any. Otherwise, we return the first reward from the other tokens
+        return (
+          this.rewards.filter(
```

Comment:
refactor the filter for `reward.amount >0`. then use that in the next to searches

Reply:
Done